### PR TITLE
Lds 5926

### PR DESCRIPTION
**Pull request by dunethomas**

### CODE REVIEW

**Comment by benbarefield** [comment]
👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ## Release History
-
+- Unreleased
+	- Capi should not throw error when a value change handler unsubscribes another change handler [LDS-5926]
```
Comment (jennifermckenzie) [inline CHANGELOG.md]: using a tab instead of spaces to indent

File: package.json [hunk 2]
```diff
@@ -29,10 +29,13 @@
...
-        "coverjs" : "0.0.11-alpha",
+        "coverjs": "0.0.11-alpha",
         "grunt": "0.4.2",
         "grunt-update-json": "0.2.1",
         "grunt-template": "0.2.3"
     },
-    "keywords": []
+    "keywords": [],
+    "dependencies": {
+        "unzip": "^0.1.6-alpha"
```
Comment (jennifermckenzie) [inline package.json]: is this being used?

Comment (dunethomas) [reply package.json]: I wonder if some automated thing wrote this in there. I dropped the keywords cause I though you were referring to that, but now I see you were talking about the unzip... I really have no idea if this is used or not. It seems to have been generated code from who-knows-where. I yanked it and it builds fine.

File: CHANGELOG.md (hunk 1 above)
Comment (dunethomas) [reply CHANGELOG.md]: crumb! Yerp... fixed

**Comment by dunethomas** [comment]
edits made. 🐔

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ## Release History
-
+- Unreleased
+    - Capi should not throw error when a value change handler unsubscribes another change handler [LDS-5926]
```
Comment (jennifermckenzie) [inline CHANGELOG.md]: Please remove the ticket ID ([LDS-5926]) from the readme (since this is a publicly available library)

**Comment by dunethomas** [comment]
Done. 🐔

**Comment by jennifermckenzie** [comment]
👍

**Pull request merged**